content/blog/2020/12/2020-12-16-call-for-mentors.adoc

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Google Summer of Code 2021 call for Project Ideas and Mentors"
+tags:
+- gsoc
+- gsoc2021
+- community
+- events
+- developer
+author: marckk
+links:
+  gitter: jenkinsci/gsoc-sig
+  googlegroup: jenkins-gsoc-all-public
+  sig: gsoc
+---
+
+NOTE: link:https://developers.google.com/open-source/gsoc/[Google Summer of Code] (GSoC)
+is as program where students are paid a stipend by Google to work on a free open source project.
+Students work on the project full-time for three months (June to August).
```

Comment:
I thought that they changed the scale of the work in 2021 to be nearer to half-time than full-time.
```suggestion
Students work on the project for three months (June to August).
```